javascripts/2048.js

```diff
@@ -1,33 +1,39 @@
 $(document).ready(function() {
   console.log('ready!');
-  var openCells = [["r1", "c1"], ["r1", "c2"], ["r1", "c3"], ["r1", "c0"],
+  var openCells = [["r1", "c1"], ["r1", "c2"], ["r1", "c0"], ["r1", "c3"],
                    ["r2", "c1"], ["r2", "c2"], ["r2", "c3"], ["r2", "c0"],
                    ["r3", "c1"], ["r3", "c2"], ["r3", "c3"], ["r3", "c0"],
                    ["r0", "c1"], ["r0", "c2"], ["r0", "c3"], ["r0", "c0"]];
-  var usedCells = []
+
+  var usedCells = [];
+
+
   newTile(openCells, usedCells);
   newTile(openCells, usedCells);
 
+
   $('body').keydown(function(event){
     var arrow_keys = [37, 38, 39, 40];
     if(arrow_keys.indexOf(event.which) > -1) {
       var tile = $('.tile');
-      // newTile(openCells, usedCells);
       playGame(tile, openCells, usedCells, event.which);
```

Comment:
In this instance, you're passing `tile` as an argument, but it doesn't seem necessary. It's a list of all the html divs that are tiles, but I don't see anywhere in your code that uses it.